jest.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
the jest config in package.json wasn't working properly, I've moved it here and fixed the path condition